osd_cluster_admin/osd_nodes/osd-nodes-machinepools-about.adoc

```diff
@@ -27,11 +27,58 @@ By default, a cluster is created with one machine pool. You can add additional m
 Multiple machine pools can exist on a single cluster, and they can each have different types or different size nodes.
 
 == Machine pools in multiple zone clusters
-When you create a machine pool in a multiple availability zone (Multi-AZ) cluster, that one machine pool has 3 zones. The machine pool, in turn, creates a total of 3 compute machine sets - one for each zone in the cluster. Each of those compute machine sets manages one or more machines in its respective availability zone.
+By default, when you create a machine pool in a multiple availability zone (Multi-AZ) cluster, that one machine pool has 3 zones. The machine pool, in turn, creates a total of 3 compute machine sets - one for each zone in the cluster. Each of those compute machine sets manages one or more machines in its respective availability zone.
 
 If you create a new Multi-AZ cluster, the machine pools are replicated to those zones automatically. If you add a machine pool to an existing Multi-AZ, the new pool is automatically created in those zones. Similarly, deleting a machine pool will delete it from all zones.
 Due to this multiplicative effect, using machine pools in Multi-AZ cluster can consume more of your project's quota for a specific region when creating machine pools.
 
+=== Deploying a machine pool in a single availability zone within a Multi-AZ cluster
+{product-title} on {GCP} users can deploy a single machine pool in a specific availability zone that is part of a Multi-AZ cluster using the {cluster-manager} CLI (`ocm`). This option is especially useful in the following situations:
+
+- Your desired instance type is not available in all the availability zones of a specific region.
+- Your cluster does not need more than one of the desired instance types.
+
+[IMPORTANT]
+====
+You cannot assign the default machine pool to a single availability zone during the provisioning of a Multi-AZ cluster.
+====
+
+.Prerequisites
+*  You installed the link:https://console.redhat.com/openshift/downloads[OpenShift Cluster Manager API command-line interface (`ocm`)].
+
+[IMPORTANT]
+====
+[subs="attributes+"]
+OpenShift Cluster Manager API command-line interface (`ocm`) is a Technology Preview feature only.
+For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+====
+
+.Procedure
+
+** Deploy a machine pool to a specific availability zone by running the following command:
++
+[source,terminal]
+----
+ocm create machine-pool \
+  --cluster <cluster_name|cluster_id> \ <1>
+  --instance-type <instance_type> \ <2>
+  --replicas <number_of_replicas> \ <3>
+  --availability-zone <availability_zone> <4>
+  --name <machine_pool_id> <5>
```

Comment:
This does not reflect the current cli command.

Reply:
This is being discussed in this thread: https://github.com/openshift/openshift-docs/pull/88722#discussion_r1967830107